purchase_order_line_description/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2015 Alex Comba - Agile Business Group
```

Comment:
I understand this is very small point. But would it be nice to use single quote in all place? I think it would be good when we search for codes, non-blocking though.